pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+markers =
+    shadow: optimization with Shadow backend (quicker simulations)
+    srw: optimization with SRW backend (slower simulations)
+    test-func: optimization on test functions (no latency)
```

Comment:
Should it have an underscore rather than `-`, as it's used in `bloptools/tests/test_bayesian_himmelblau.py` (`@pytest.mark.test_func`)? Or does pytest understand and convert them automatically?
```suggestion
    test_func: optimization on test functions (no latency)
```

Reply:
good point, let's make them all with a `-`

Reply:
oh flake really doesn't like that